<?xml version="1.0" encoding="UTF-8"?>
<testng-results>
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="11778" started-at="2015-06-19T15:35:02Z" finished-at="2015-06-19T15:35:14Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="11778" started-at="2015-06-19T15:35:02Z" finished-at="2015-06-19T15:35:14Z">
      <class name="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest">
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="5" started-at="2015-06-19T15:35:02Z" finished-at="2015-06-19T15:35:02Z">
        </test-method>
        <test-method status="PASS" signature="mustImmediatelyPassOnOnErrorEventsReceivedFromItsUpstreamToItsDownstream()" name="mustImmediatelyPassOnOnErrorEventsReceivedFromItsUpstreamToItsDownstream" duration-ms="99" started-at="2015-06-19T15:35:02Z" finished-at="2015-06-19T15:35:02Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="1" started-at="2015-06-19T15:35:02Z" finished-at="2015-06-19T15:35:02Z">
        </test-method>
        <test-method status="SKIP" signature="optional_spec104_mustSignalOnErrorWhenFails()" name="optional_spec104_mustSignalOnErrorWhenFails" duration-ms="2" started-at="2015-06-19T15:35:02Z" finished-at="2015-06-19T15:35:02Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping because no error state Publisher provided, and the test requires it. Please implement PublisherVerification#createFailedPublisher to run this test.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping because no error state Publisher provided, and the test requires it. Please implement PublisherVerification#createFailedPublisher to run this test.
	at org.reactivestreams.tck.PublisherVerification.potentiallyPendingTest(PublisherVerification.java:1147)
	at org.reactivestreams.tck.PublisherVerification.whenHasErrorPublisherTest(PublisherVerification.java:1136)
	at org.reactivestreams.tck.PublisherVerification.optional_spec104_mustSignalOnErrorWhenFails(PublisherVerification.java:346)
	at org.reactivestreams.tck.IdentityProcessorVerification.optional_spec104_mustSignalOnErrorWhenFails(IdentityProcessorVerification.java:240)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:02Z" finished-at="2015-06-19T15:35:02Z">
        </test-method>
        <test-method status="PASS" signature="optional_spec105_emptyStreamMustTerminateBySignallingOnComplete()" name="optional_spec105_emptyStreamMustTerminateBySignallingOnComplete" duration-ms="259" started-at="2015-06-19T15:35:02Z" finished-at="2015-06-19T15:35:03Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:03Z" finished-at="2015-06-19T15:35:03Z">
        </test-method>
        <test-method status="SKIP" signature="optional_spec111_maySupportMultiSubscribe()" name="optional_spec111_maySupportMultiSubscribe" duration-ms="255" started-at="2015-06-19T15:35:03Z" finished-at="2015-06-19T15:35:03Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipped because tested publisher does NOT implement this OPTIONAL requirement. Reason for skipping was: Could not subscribe org.reactivestreams.tck.TestEnvironment$ManualSubscriberWithSubscriptionSupport@6e1567f1 to Publisher com.aol.simple.react.reactivestreams.FutureStreamProcessor@59494225 within 250 ms]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipped because tested publisher does NOT implement this OPTIONAL requirement. Reason for skipping was: Could not subscribe org.reactivestreams.tck.TestEnvironment$ManualSubscriberWithSubscriptionSupport@6e1567f1 to Publisher com.aol.simple.react.reactivestreams.FutureStreamProcessor@59494225 within 250 ms
	at org.reactivestreams.tck.PublisherVerification.notVerified(PublisherVerification.java:1180)
	at org.reactivestreams.tck.PublisherVerification.optionalActivePublisherTest(PublisherVerification.java:1121)
	at org.reactivestreams.tck.PublisherVerification.optional_spec111_maySupportMultiSubscribe(PublisherVerification.java:543)
	at org.reactivestreams.tck.IdentityProcessorVerification.optional_spec111_maySupportMultiSubscribe(IdentityProcessorVerification.java:300)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:03Z" finished-at="2015-06-19T15:35:03Z">
        </test-method>
        <test-method status="SKIP" signature="optional_spec111_multicast_mustProduceTheSameElementsInTheSameSequenceToAllOfItsSubscribersWhenRequestingManyUpfront()" name="optional_spec111_multicast_mustProduceTheSameElementsInTheSameSequenceToAllOfItsSubscribersWhenRequestingManyUpfront" duration-ms="252" started-at="2015-06-19T15:35:03Z" finished-at="2015-06-19T15:35:03Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipped because tested publisher does NOT implement this OPTIONAL requirement. Reason for skipping was: Could not subscribe org.reactivestreams.tck.TestEnvironment$ManualSubscriberWithSubscriptionSupport@4b952a2d to Publisher com.aol.simple.react.reactivestreams.FutureStreamProcessor@69ea3742 within 250 ms]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipped because tested publisher does NOT implement this OPTIONAL requirement. Reason for skipping was: Could not subscribe org.reactivestreams.tck.TestEnvironment$ManualSubscriberWithSubscriptionSupport@4b952a2d to Publisher com.aol.simple.react.reactivestreams.FutureStreamProcessor@69ea3742 within 250 ms
	at org.reactivestreams.tck.PublisherVerification.notVerified(PublisherVerification.java:1180)
	at org.reactivestreams.tck.PublisherVerification.optionalActivePublisherTest(PublisherVerification.java:1121)
	at org.reactivestreams.tck.PublisherVerification.optional_spec111_multicast_mustProduceTheSameElementsInTheSameSequenceToAllOfItsSubscribersWhenRequestingManyUpfront(PublisherVerification.java:608)
	at org.reactivestreams.tck.IdentityProcessorVerification.optional_spec111_multicast_mustProduceTheSameElementsInTheSameSequenceToAllOfItsSubscribersWhenRequestingManyUpfront(IdentityProcessorVerification.java:310)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="SKIP" signature="optional_spec111_multicast_mustProduceTheSameElementsInTheSameSequenceToAllOfItsSubscribersWhenRequestingManyUpfrontAndCompleteAsExpected()" name="optional_spec111_multicast_mustProduceTheSameElementsInTheSameSequenceToAllOfItsSubscribersWhenRequestingManyUpfrontAndCompleteAsExpected" duration-ms="257" started-at="2015-06-19T15:35:03Z" finished-at="2015-06-19T15:35:03Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipped because tested publisher does NOT implement this OPTIONAL requirement. Reason for skipping was: Could not subscribe org.reactivestreams.tck.TestEnvironment$ManualSubscriberWithSubscriptionSupport@49993335 to Publisher com.aol.simple.react.reactivestreams.FutureStreamProcessor@5025a98f within 250 ms]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipped because tested publisher does NOT implement this OPTIONAL requirement. Reason for skipping was: Could not subscribe org.reactivestreams.tck.TestEnvironment$ManualSubscriberWithSubscriptionSupport@49993335 to Publisher com.aol.simple.react.reactivestreams.FutureStreamProcessor@5025a98f within 250 ms
	at org.reactivestreams.tck.PublisherVerification.notVerified(PublisherVerification.java:1180)
	at org.reactivestreams.tck.PublisherVerification.optionalActivePublisherTest(PublisherVerification.java:1121)
	at org.reactivestreams.tck.PublisherVerification.optional_spec111_multicast_mustProduceTheSameElementsInTheSameSequenceToAllOfItsSubscribersWhenRequestingManyUpfrontAndCompleteAsExpected(PublisherVerification.java:641)
	at org.reactivestreams.tck.IdentityProcessorVerification.optional_spec111_multicast_mustProduceTheSameElementsInTheSameSequenceToAllOfItsSubscribersWhenRequestingManyUpfrontAndCompleteAsExpected(IdentityProcessorVerification.java:315)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:03Z" finished-at="2015-06-19T15:35:03Z">
        </test-method>
        <test-method status="SKIP" signature="optional_spec111_multicast_mustProduceTheSameElementsInTheSameSequenceToAllOfItsSubscribersWhenRequestingOneByOne()" name="optional_spec111_multicast_mustProduceTheSameElementsInTheSameSequenceToAllOfItsSubscribersWhenRequestingOneByOne" duration-ms="257" started-at="2015-06-19T15:35:03Z" finished-at="2015-06-19T15:35:04Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipped because tested publisher does NOT implement this OPTIONAL requirement. Reason for skipping was: Could not subscribe org.reactivestreams.tck.TestEnvironment$ManualSubscriberWithSubscriptionSupport@6e1ec318 to Publisher com.aol.simple.react.reactivestreams.FutureStreamProcessor@10bdf5e5 within 250 ms]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipped because tested publisher does NOT implement this OPTIONAL requirement. Reason for skipping was: Could not subscribe org.reactivestreams.tck.TestEnvironment$ManualSubscriberWithSubscriptionSupport@6e1ec318 to Publisher com.aol.simple.react.reactivestreams.FutureStreamProcessor@10bdf5e5 within 250 ms
	at org.reactivestreams.tck.PublisherVerification.notVerified(PublisherVerification.java:1180)
	at org.reactivestreams.tck.PublisherVerification.optionalActivePublisherTest(PublisherVerification.java:1121)
	at org.reactivestreams.tck.PublisherVerification.optional_spec111_multicast_mustProduceTheSameElementsInTheSameSequenceToAllOfItsSubscribersWhenRequestingOneByOne(PublisherVerification.java:557)
	at org.reactivestreams.tck.IdentityProcessorVerification.optional_spec111_multicast_mustProduceTheSameElementsInTheSameSequenceToAllOfItsSubscribersWhenRequestingOneByOne(IdentityProcessorVerification.java:305)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:03Z" finished-at="2015-06-19T15:35:03Z">
        </test-method>
        <test-method status="PASS" signature="required_createPublisher1MustProduceAStreamOfExactly1Element()" name="required_createPublisher1MustProduceAStreamOfExactly1Element" duration-ms="22" started-at="2015-06-19T15:35:04Z" finished-at="2015-06-19T15:35:04Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:04Z" finished-at="2015-06-19T15:35:04Z">
        </test-method>
        <test-method status="PASS" signature="required_createPublisher3MustProduceAStreamOfExactly3Elements()" name="required_createPublisher3MustProduceAStreamOfExactly3Elements" duration-ms="5" started-at="2015-06-19T15:35:04Z" finished-at="2015-06-19T15:35:04Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:04Z" finished-at="2015-06-19T15:35:04Z">
        </test-method>
        <test-method status="FAIL" signature="required_exerciseWhiteboxHappyPath()" name="required_exerciseWhiteboxHappyPath" duration-ms="267" started-at="2015-06-19T15:35:04Z" finished-at="2015-06-19T15:35:04Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Subscriber com.aol.simple.react.reactivestreams.FutureStreamProcessor@5f9d02cb did not call `registerOnNext(5)` within 250 ms]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Subscriber com.aol.simple.react.reactivestreams.FutureStreamProcessor@5f9d02cb did not call `registerOnNext(5)` within 250 ms
	at org.testng.Assert.fail(Assert.java:78)
	at org.reactivestreams.tck.TestEnvironment.flopAndFail(TestEnvironment.java:180)
	at org.reactivestreams.tck.TestEnvironment$Receptacle.next(TestEnvironment.java:878)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification$BlackboxProbe.expectNext(SubscriberWhiteboxVerification.java:655)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification$BlackboxProbe.expectNext(SubscriberWhiteboxVerification.java:651)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification$1.run(SubscriberWhiteboxVerification.java:78)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.subscriberTest(SubscriberWhiteboxVerification.java:486)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.required_exerciseWhiteboxHappyPath(SubscriberWhiteboxVerification.java:69)
	at org.reactivestreams.tck.IdentityProcessorVerification.required_exerciseWhiteboxHappyPath(IdentityProcessorVerification.java:492)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
Caused by: java.lang.AssertionError: Subscriber com.aol.simple.react.reactivestreams.FutureStreamProcessor@5f9d02cb did not call `registerOnNext(5)` within 250 ms
	at org.testng.Assert.fail(Assert.java:89)
	at org.reactivestreams.tck.TestEnvironment.flopAndFail(TestEnvironment.java:177)
	... 32 more
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:04Z" finished-at="2015-06-19T15:35:04Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:04Z" finished-at="2015-06-19T15:35:04Z">
        </test-method>
        <test-method status="FAIL" signature="required_mustRequestFromUpstreamForElementsThatHaveBeenRequestedLongAgo()" name="required_mustRequestFromUpstreamForElementsThatHaveBeenRequestedLongAgo" duration-ms="255" started-at="2015-06-19T15:35:04Z" finished-at="2015-06-19T15:35:04Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[timeout while awaiting 5 within 250 ms]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: timeout while awaiting 5 within 250 ms
	at org.testng.Assert.fail(Assert.java:78)
	at org.reactivestreams.tck.TestEnvironment.flopAndFail(TestEnvironment.java:180)
	at org.reactivestreams.tck.TestEnvironment$Receptacle.next(TestEnvironment.java:878)
	at org.reactivestreams.tck.TestEnvironment$ManualSubscriber.nextElement(TestEnvironment.java:400)
	at org.reactivestreams.tck.TestEnvironment$ManualSubscriber.nextElement(TestEnvironment.java:396)
	at org.reactivestreams.tck.IdentityProcessorVerification$TestSetup.expectNextElement(IdentityProcessorVerification.java:724)
	at org.reactivestreams.tck.IdentityProcessorVerification$6$1.<init>(IdentityProcessorVerification.java:638)
	at org.reactivestreams.tck.IdentityProcessorVerification$6.apply(IdentityProcessorVerification.java:632)
	at org.reactivestreams.tck.IdentityProcessorVerification$6.apply(IdentityProcessorVerification.java:629)
	at org.reactivestreams.tck.IdentityProcessorVerification.optionalMultipleSubscribersTest(IdentityProcessorVerification.java:694)
	at org.reactivestreams.tck.IdentityProcessorVerification.required_mustRequestFromUpstreamForElementsThatHaveBeenRequestedLongAgo(IdentityProcessorVerification.java:629)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
Caused by: java.lang.AssertionError: timeout while awaiting 5 within 250 ms
	at org.testng.Assert.fail(Assert.java:89)
	at org.reactivestreams.tck.TestEnvironment.flopAndFail(TestEnvironment.java:177)
	... 34 more
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="PASS" signature="required_spec101_subscriptionRequestMustResultInTheCorrectNumberOfProducedElements()" name="required_spec101_subscriptionRequestMustResultInTheCorrectNumberOfProducedElements" duration-ms="761" started-at="2015-06-19T15:35:04Z" finished-at="2015-06-19T15:35:05Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:04Z" finished-at="2015-06-19T15:35:04Z">
        </test-method>
        <test-method status="PASS" signature="required_spec102_maySignalLessThanRequestedAndTerminateSubscription()" name="required_spec102_maySignalLessThanRequestedAndTerminateSubscription" duration-ms="3" started-at="2015-06-19T15:35:05Z" finished-at="2015-06-19T15:35:05Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:05Z" finished-at="2015-06-19T15:35:05Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:05Z" finished-at="2015-06-19T15:35:05Z">
        </test-method>
        <test-method status="FAIL" signature="required_spec104_mustCallOnErrorOnAllItsSubscribersIfItEncountersANonRecoverableError()" name="required_spec104_mustCallOnErrorOnAllItsSubscribersIfItEncountersANonRecoverableError" duration-ms="254" started-at="2015-06-19T15:35:05Z" finished-at="2015-06-19T15:35:05Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Could not subscribe org.reactivestreams.tck.IdentityProcessorVerification$ManualSubscriberWithErrorCollection@35d176f7 to Publisher com.aol.simple.react.reactivestreams.FutureStreamProcessor@551aa95a within 250 ms]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Could not subscribe org.reactivestreams.tck.IdentityProcessorVerification$ManualSubscriberWithErrorCollection@35d176f7 to Publisher com.aol.simple.react.reactivestreams.FutureStreamProcessor@551aa95a within 250 ms
	at org.testng.Assert.fail(Assert.java:89)
	at org.reactivestreams.tck.TestEnvironment.flop(TestEnvironment.java:117)
	at org.reactivestreams.tck.TestEnvironment$Promise.expectCompletion(TestEnvironment.java:839)
	at org.reactivestreams.tck.TestEnvironment.subscribe(TestEnvironment.java:193)
	at org.reactivestreams.tck.TestEnvironment.subscribe(TestEnvironment.java:188)
	at org.reactivestreams.tck.IdentityProcessorVerification$3$1.<init>(IdentityProcessorVerification.java:395)
	at org.reactivestreams.tck.IdentityProcessorVerification$3.apply(IdentityProcessorVerification.java:390)
	at org.reactivestreams.tck.IdentityProcessorVerification$3.apply(IdentityProcessorVerification.java:387)
	at org.reactivestreams.tck.IdentityProcessorVerification.optionalMultipleSubscribersTest(IdentityProcessorVerification.java:694)
	at org.reactivestreams.tck.IdentityProcessorVerification.required_spec104_mustCallOnErrorOnAllItsSubscribersIfItEncountersANonRecoverableError(IdentityProcessorVerification.java:387)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:05Z" finished-at="2015-06-19T15:35:05Z">
        </test-method>
        <test-method status="PASS" signature="required_spec105_mustSignalOnCompleteWhenFiniteStreamTerminates()" name="required_spec105_mustSignalOnCompleteWhenFiniteStreamTerminates" duration-ms="256" started-at="2015-06-19T15:35:05Z" finished-at="2015-06-19T15:35:05Z">
        </test-method>
        <test-method status="PASS" signature="required_spec107_mustNotEmitFurtherSignalsOnceOnCompleteHasBeenSignalled()" name="required_spec107_mustNotEmitFurtherSignalsOnceOnCompleteHasBeenSignalled" duration-ms="258" started-at="2015-06-19T15:35:05Z" finished-at="2015-06-19T15:35:06Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:05Z" finished-at="2015-06-19T15:35:05Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:06Z" finished-at="2015-06-19T15:35:06Z">
        </test-method>
        <test-method status="SKIP" signature="required_spec109_mayRejectCallsToSubscribeIfPublisherIsUnableOrUnwillingToServeThemRejectionMustTriggerOnErrorAfterOnSubscribe()" name="required_spec109_mayRejectCallsToSubscribeIfPublisherIsUnableOrUnwillingToServeThemRejectionMustTriggerOnErrorAfterOnSubscribe" duration-ms="1" started-at="2015-06-19T15:35:06Z" finished-at="2015-06-19T15:35:06Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping because no error state Publisher provided, and the test requires it. Please implement PublisherVerification#createFailedPublisher to run this test.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping because no error state Publisher provided, and the test requires it. Please implement PublisherVerification#createFailedPublisher to run this test.
	at org.reactivestreams.tck.PublisherVerification.potentiallyPendingTest(PublisherVerification.java:1147)
	at org.reactivestreams.tck.PublisherVerification.whenHasErrorPublisherTest(PublisherVerification.java:1136)
	at org.reactivestreams.tck.PublisherVerification.required_spec109_mayRejectCallsToSubscribeIfPublisherIsUnableOrUnwillingToServeThemRejectionMustTriggerOnErrorAfterOnSubscribe(PublisherVerification.java:506)
	at org.reactivestreams.tck.IdentityProcessorVerification.required_spec109_mayRejectCallsToSubscribeIfPublisherIsUnableOrUnwillingToServeThemRejectionMustTriggerOnErrorAfterOnSubscribe(IdentityProcessorVerification.java:285)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="PASS" signature="required_spec109_mustIssueOnSubscribeForNonNullSubscriber()" name="required_spec109_mustIssueOnSubscribeForNonNullSubscriber" duration-ms="2" started-at="2015-06-19T15:35:06Z" finished-at="2015-06-19T15:35:06Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:06Z" finished-at="2015-06-19T15:35:06Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="1" started-at="2015-06-19T15:35:06Z" finished-at="2015-06-19T15:35:06Z">
        </test-method>
        <test-method status="PASS" signature="required_spec109_subscribeThrowNPEOnNullSubscriber()" name="required_spec109_subscribeThrowNPEOnNullSubscriber" duration-ms="0" started-at="2015-06-19T15:35:06Z" finished-at="2015-06-19T15:35:06Z">
        </test-method>
        <test-method status="PASS" signature="required_spec201_mustSignalDemandViaSubscriptionRequest()" name="required_spec201_mustSignalDemandViaSubscriptionRequest" duration-ms="2" started-at="2015-06-19T15:35:06Z" finished-at="2015-06-19T15:35:06Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:06Z" finished-at="2015-06-19T15:35:06Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:06Z" finished-at="2015-06-19T15:35:06Z">
        </test-method>
        <test-method status="PASS" signature="required_spec203_mustNotCallMethodsOnSubscriptionOrPublisherInOnComplete()" name="required_spec203_mustNotCallMethodsOnSubscriptionOrPublisherInOnComplete" duration-ms="2" started-at="2015-06-19T15:35:06Z" finished-at="2015-06-19T15:35:06Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:06Z" finished-at="2015-06-19T15:35:06Z">
        </test-method>
        <test-method status="PASS" signature="required_spec203_mustNotCallMethodsOnSubscriptionOrPublisherInOnError()" name="required_spec203_mustNotCallMethodsOnSubscriptionOrPublisherInOnError" duration-ms="2" started-at="2015-06-19T15:35:06Z" finished-at="2015-06-19T15:35:06Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="1" started-at="2015-06-19T15:35:06Z" finished-at="2015-06-19T15:35:06Z">
        </test-method>
        <test-method status="PASS" signature="required_spec205_mustCallSubscriptionCancelIfItAlreadyHasAnSubscriptionAndReceivesAnotherOnSubscribeSignal()" name="required_spec205_mustCallSubscriptionCancelIfItAlreadyHasAnSubscriptionAndReceivesAnotherOnSubscribeSignal" duration-ms="255" started-at="2015-06-19T15:35:06Z" finished-at="2015-06-19T15:35:06Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:06Z" finished-at="2015-06-19T15:35:06Z">
        </test-method>
        <test-method status="PASS" signature="required_spec208_mustBePreparedToReceiveOnNextSignalsAfterHavingCalledSubscriptionCancel()" name="required_spec208_mustBePreparedToReceiveOnNextSignalsAfterHavingCalledSubscriptionCancel" duration-ms="258" started-at="2015-06-19T15:35:06Z" finished-at="2015-06-19T15:35:06Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:06Z" finished-at="2015-06-19T15:35:06Z">
        </test-method>
        <test-method status="PASS" signature="required_spec209_mustBePreparedToReceiveAnOnCompleteSignalWithPrecedingRequestCall()" name="required_spec209_mustBePreparedToReceiveAnOnCompleteSignalWithPrecedingRequestCall" duration-ms="254" started-at="2015-06-19T15:35:06Z" finished-at="2015-06-19T15:35:06Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:06Z" finished-at="2015-06-19T15:35:06Z">
        </test-method>
        <test-method status="FAIL" signature="required_spec209_mustBePreparedToReceiveAnOnCompleteSignalWithoutPrecedingRequestCall()" name="required_spec209_mustBePreparedToReceiveAnOnCompleteSignalWithoutPrecedingRequestCall" duration-ms="255" started-at="2015-06-19T15:35:06Z" finished-at="2015-06-19T15:35:07Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Subscriber com.aol.simple.react.reactivestreams.FutureStreamProcessor@3dfc5fb8 did not call `registerOnComplete()` within 250 ms]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Subscriber com.aol.simple.react.reactivestreams.FutureStreamProcessor@3dfc5fb8 did not call `registerOnComplete()` within 250 ms
	at org.testng.Assert.fail(Assert.java:89)
	at org.reactivestreams.tck.TestEnvironment.flop(TestEnvironment.java:117)
	at org.reactivestreams.tck.TestEnvironment$Receptacle.expectCompletion(TestEnvironment.java:919)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification$BlackboxProbe.expectCompletion(SubscriberWhiteboxVerification.java:674)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification$BlackboxProbe.expectCompletion(SubscriberWhiteboxVerification.java:670)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification$BlackboxProbe.expectCompletion(SubscriberWhiteboxVerification.java:666)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification$8.run(SubscriberWhiteboxVerification.java:290)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.subscriberTest(SubscriberWhiteboxVerification.java:486)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.required_spec209_mustBePreparedToReceiveAnOnCompleteSignalWithoutPrecedingRequestCall(SubscriberWhiteboxVerification.java:286)
	at org.reactivestreams.tck.IdentityProcessorVerification.required_spec209_mustBePreparedToReceiveAnOnCompleteSignalWithoutPrecedingRequestCall(IdentityProcessorVerification.java:547)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="1" started-at="2015-06-19T15:35:07Z" finished-at="2015-06-19T15:35:07Z">
        </test-method>
        <test-method status="PASS" signature="required_spec210_mustBePreparedToReceiveAnOnErrorSignalWithPrecedingRequestCall()" name="required_spec210_mustBePreparedToReceiveAnOnErrorSignalWithPrecedingRequestCall" duration-ms="1" started-at="2015-06-19T15:35:07Z" finished-at="2015-06-19T15:35:07Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:07Z" finished-at="2015-06-19T15:35:07Z">
        </test-method>
        <test-method status="PASS" signature="required_spec210_mustBePreparedToReceiveAnOnErrorSignalWithoutPrecedingRequestCall()" name="required_spec210_mustBePreparedToReceiveAnOnErrorSignalWithoutPrecedingRequestCall" duration-ms="1" started-at="2015-06-19T15:35:07Z" finished-at="2015-06-19T15:35:07Z">
        </test-method>
        <test-method status="PASS" signature="required_spec213_onError_mustThrowNullPointerExceptionWhenParametersAreNull()" name="required_spec213_onError_mustThrowNullPointerExceptionWhenParametersAreNull" duration-ms="2" started-at="2015-06-19T15:35:07Z" finished-at="2015-06-19T15:35:07Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:07Z" finished-at="2015-06-19T15:35:07Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:07Z" finished-at="2015-06-19T15:35:07Z">
        </test-method>
        <test-method status="PASS" signature="required_spec213_onNext_mustThrowNullPointerExceptionWhenParametersAreNull()" name="required_spec213_onNext_mustThrowNullPointerExceptionWhenParametersAreNull" duration-ms="1" started-at="2015-06-19T15:35:07Z" finished-at="2015-06-19T15:35:07Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="1" started-at="2015-06-19T15:35:07Z" finished-at="2015-06-19T15:35:07Z">
        </test-method>
        <test-method status="PASS" signature="required_spec213_onSubscribe_mustThrowNullPointerExceptionWhenParametersAreNull()" name="required_spec213_onSubscribe_mustThrowNullPointerExceptionWhenParametersAreNull" duration-ms="1" started-at="2015-06-19T15:35:07Z" finished-at="2015-06-19T15:35:07Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:07Z" finished-at="2015-06-19T15:35:07Z">
        </test-method>
        <test-method status="PASS" signature="required_spec302_mustAllowSynchronousRequestCallsFromOnNextAndOnSubscribe()" name="required_spec302_mustAllowSynchronousRequestCallsFromOnNextAndOnSubscribe" duration-ms="256" started-at="2015-06-19T15:35:07Z" finished-at="2015-06-19T15:35:07Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:07Z" finished-at="2015-06-19T15:35:07Z">
        </test-method>
        <test-method status="PASS" signature="required_spec303_mustNotAllowUnboundedRecursion()" name="required_spec303_mustNotAllowUnboundedRecursion" duration-ms="29" started-at="2015-06-19T15:35:07Z" finished-at="2015-06-19T15:35:07Z">
        </test-method>
        <test-method status="PASS" signature="required_spec306_afterSubscriptionIsCancelledRequestMustBeNops()" name="required_spec306_afterSubscriptionIsCancelledRequestMustBeNops" duration-ms="254" started-at="2015-06-19T15:35:07Z" finished-at="2015-06-19T15:35:07Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:07Z" finished-at="2015-06-19T15:35:07Z">
        </test-method>
        <test-method status="PASS" signature="required_spec307_afterSubscriptionIsCancelledAdditionalCancelationsMustBeNops()" name="required_spec307_afterSubscriptionIsCancelledAdditionalCancelationsMustBeNops" duration-ms="257" started-at="2015-06-19T15:35:07Z" finished-at="2015-06-19T15:35:07Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:07Z" finished-at="2015-06-19T15:35:07Z">
        </test-method>
        <test-method status="FAIL" signature="required_spec308_requestMustRegisterGivenNumberElementsToBeProduced()" name="required_spec308_requestMustRegisterGivenNumberElementsToBeProduced" duration-ms="255" started-at="2015-06-19T15:35:07Z" finished-at="2015-06-19T15:35:08Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Subscriber com.aol.simple.react.reactivestreams.FutureStreamProcessor@23fe1d71 did not call `registerOnNext(5)` within 250 ms]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Subscriber com.aol.simple.react.reactivestreams.FutureStreamProcessor@23fe1d71 did not call `registerOnNext(5)` within 250 ms
	at org.testng.Assert.fail(Assert.java:78)
	at org.reactivestreams.tck.TestEnvironment.flopAndFail(TestEnvironment.java:180)
	at org.reactivestreams.tck.TestEnvironment$Receptacle.next(TestEnvironment.java:878)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification$BlackboxProbe.expectNext(SubscriberWhiteboxVerification.java:655)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification$BlackboxProbe.expectNext(SubscriberWhiteboxVerification.java:651)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification$14.run(SubscriberWhiteboxVerification.java:428)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.subscriberTest(SubscriberWhiteboxVerification.java:486)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.required_spec308_requestMustRegisterGivenNumberElementsToBeProduced(SubscriberWhiteboxVerification.java:424)
	at org.reactivestreams.tck.IdentityProcessorVerification.required_spec308_requestMustRegisterGivenNumberElementsToBeProduced(IdentityProcessorVerification.java:595)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
Caused by: java.lang.AssertionError: Subscriber com.aol.simple.react.reactivestreams.FutureStreamProcessor@23fe1d71 did not call `registerOnNext(5)` within 250 ms
	at org.testng.Assert.fail(Assert.java:89)
	at org.reactivestreams.tck.TestEnvironment.flopAndFail(TestEnvironment.java:177)
	... 32 more
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:07Z" finished-at="2015-06-19T15:35:07Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:08Z" finished-at="2015-06-19T15:35:08Z">
        </test-method>
        <test-method status="PASS" signature="required_spec309_requestNegativeNumberMustSignalIllegalArgumentException()" name="required_spec309_requestNegativeNumberMustSignalIllegalArgumentException" duration-ms="253" started-at="2015-06-19T15:35:08Z" finished-at="2015-06-19T15:35:08Z">
        </test-method>
        <test-method status="PASS" signature="required_spec309_requestZeroMustSignalIllegalArgumentException()" name="required_spec309_requestZeroMustSignalIllegalArgumentException" duration-ms="258" started-at="2015-06-19T15:35:08Z" finished-at="2015-06-19T15:35:08Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:08Z" finished-at="2015-06-19T15:35:08Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="1" started-at="2015-06-19T15:35:08Z" finished-at="2015-06-19T15:35:08Z">
        </test-method>
        <test-method status="PASS" signature="required_spec312_cancelMustMakeThePublisherToEventuallyStopSignaling()" name="required_spec312_cancelMustMakeThePublisherToEventuallyStopSignaling" duration-ms="507" started-at="2015-06-19T15:35:08Z" finished-at="2015-06-19T15:35:09Z">
        </test-method>
        <test-method status="PASS" signature="required_spec313_cancelMustMakeThePublisherEventuallyDropAllReferencesToTheSubscriber()" name="required_spec313_cancelMustMakeThePublisherEventuallyDropAllReferencesToTheSubscriber" duration-ms="3526" started-at="2015-06-19T15:35:09Z" finished-at="2015-06-19T15:35:12Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:09Z" finished-at="2015-06-19T15:35:09Z">
        </test-method>
        <test-method status="PASS" signature="required_spec317_mustNotSignalOnErrorWhenPendingAboveLongMaxValue()" name="required_spec317_mustNotSignalOnErrorWhenPendingAboveLongMaxValue" duration-ms="557" started-at="2015-06-19T15:35:12Z" finished-at="2015-06-19T15:35:13Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:12Z" finished-at="2015-06-19T15:35:12Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:13Z" finished-at="2015-06-19T15:35:13Z">
        </test-method>
        <test-method status="PASS" signature="required_spec317_mustSupportACumulativePendingElementCountUpToLongMaxValue()" name="required_spec317_mustSupportACumulativePendingElementCountUpToLongMaxValue" duration-ms="4" started-at="2015-06-19T15:35:13Z" finished-at="2015-06-19T15:35:13Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="1" started-at="2015-06-19T15:35:13Z" finished-at="2015-06-19T15:35:13Z">
        </test-method>
        <test-method status="PASS" signature="required_spec317_mustSupportAPendingElementCountUpToLongMaxValue()" name="required_spec317_mustSupportAPendingElementCountUpToLongMaxValue" duration-ms="33" started-at="2015-06-19T15:35:13Z" finished-at="2015-06-19T15:35:13Z">
        </test-method>
        <test-method status="PASS" signature="required_validate_boundedDepthOfOnNextAndRequestRecursion()" name="required_validate_boundedDepthOfOnNextAndRequestRecursion" duration-ms="1" started-at="2015-06-19T15:35:13Z" finished-at="2015-06-19T15:35:13Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:13Z" finished-at="2015-06-19T15:35:13Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:13Z" finished-at="2015-06-19T15:35:13Z">
        </test-method>
        <test-method status="PASS" signature="required_validate_maxElementsFromPublisher()" name="required_validate_maxElementsFromPublisher" duration-ms="1" started-at="2015-06-19T15:35:13Z" finished-at="2015-06-19T15:35:13Z">
        </test-method>
        <test-method status="PASS" signature="stochastic_spec103_mustSignalOnMethodsSequentially()" name="stochastic_spec103_mustSignalOnMethodsSequentially" duration-ms="979" started-at="2015-06-19T15:35:13Z" finished-at="2015-06-19T15:35:14Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:13Z" finished-at="2015-06-19T15:35:13Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
        </test-method>
        <test-method status="SKIP" signature="untested_spec106_mustConsiderSubscriptionCancelledAfterOnErrorOrOnCompleteHasBeenCalled()" name="untested_spec106_mustConsiderSubscriptionCancelledAfterOnErrorOrOnCompleteHasBeenCalled" duration-ms="1" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Not verified by this TCK.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Not verified by this TCK.
	at org.reactivestreams.tck.PublisherVerification.notVerified(PublisherVerification.java:1166)
	at org.reactivestreams.tck.PublisherVerification.untested_spec106_mustConsiderSubscriptionCancelledAfterOnErrorOrOnCompleteHasBeenCalled(PublisherVerification.java:413)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec106_mustConsiderSubscriptionCancelledAfterOnErrorOrOnCompleteHasBeenCalled(IdentityProcessorVerification.java:255)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="SKIP" signature="untested_spec107_mustNotEmitFurtherSignalsOnceOnErrorHasBeenSignalled()" name="untested_spec107_mustNotEmitFurtherSignalsOnceOnErrorHasBeenSignalled" duration-ms="0" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Not verified by this TCK.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Not verified by this TCK.
	at org.reactivestreams.tck.PublisherVerification.notVerified(PublisherVerification.java:1166)
	at org.reactivestreams.tck.PublisherVerification.untested_spec107_mustNotEmitFurtherSignalsOnceOnErrorHasBeenSignalled(PublisherVerification.java:436)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec107_mustNotEmitFurtherSignalsOnceOnErrorHasBeenSignalled(IdentityProcessorVerification.java:265)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
        </test-method>
        <test-method status="SKIP" signature="untested_spec108_possiblyCanceledSubscriptionShouldNotReceiveOnErrorOrOnCompleteSignals()" name="untested_spec108_possiblyCanceledSubscriptionShouldNotReceiveOnErrorOrOnCompleteSignals" duration-ms="0" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Not verified by this TCK.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Not verified by this TCK.
	at org.reactivestreams.tck.PublisherVerification.notVerified(PublisherVerification.java:1166)
	at org.reactivestreams.tck.PublisherVerification.untested_spec108_possiblyCanceledSubscriptionShouldNotReceiveOnErrorOrOnCompleteSignals(PublisherVerification.java:442)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec108_possiblyCanceledSubscriptionShouldNotReceiveOnErrorOrOnCompleteSignals(IdentityProcessorVerification.java:270)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
        </test-method>
        <test-method status="SKIP" signature="untested_spec109_subscribeShouldNotThrowNonFatalThrowable()" name="untested_spec109_subscribeShouldNotThrowNonFatalThrowable" duration-ms="0" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Not verified by this TCK.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Not verified by this TCK.
	at org.reactivestreams.tck.PublisherVerification.notVerified(PublisherVerification.java:1166)
	at org.reactivestreams.tck.PublisherVerification.untested_spec109_subscribeShouldNotThrowNonFatalThrowable(PublisherVerification.java:448)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec109_subscribeShouldNotThrowNonFatalThrowable(IdentityProcessorVerification.java:275)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="SKIP" signature="untested_spec110_rejectASubscriptionRequestIfTheSameSubscriberSubscribesTwice()" name="untested_spec110_rejectASubscriptionRequestIfTheSameSubscriberSubscribesTwice" duration-ms="0" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Not verified by this TCK.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Not verified by this TCK.
	at org.reactivestreams.tck.PublisherVerification.notVerified(PublisherVerification.java:1166)
	at org.reactivestreams.tck.PublisherVerification.untested_spec110_rejectASubscriptionRequestIfTheSameSubscriberSubscribesTwice(PublisherVerification.java:537)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec110_rejectASubscriptionRequestIfTheSameSubscriberSubscribesTwice(IdentityProcessorVerification.java:295)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
        </test-method>
        <test-method status="SKIP" signature="untested_spec202_shouldAsynchronouslyDispatch()" name="untested_spec202_shouldAsynchronouslyDispatch" duration-ms="0" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Not verified using this TCK.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec202_shouldAsynchronouslyDispatch(SubscriberWhiteboxVerification.java:115)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec202_shouldAsynchronouslyDispatch(IdentityProcessorVerification.java:502)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
        </test-method>
        <test-method status="SKIP" signature="untested_spec204_mustConsiderTheSubscriptionAsCancelledInAfterRecievingOnCompleteOrOnError()" name="untested_spec204_mustConsiderTheSubscriptionAsCancelledInAfterRecievingOnCompleteOrOnError" duration-ms="0" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Not verified using this TCK.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec204_mustConsiderTheSubscriptionAsCancelledInAfterRecievingOnCompleteOrOnError(SubscriberWhiteboxVerification.java:201)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec204_mustConsiderTheSubscriptionAsCancelledInAfterRecievingOnCompleteOrOnError(IdentityProcessorVerification.java:517)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="1" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
        </test-method>
        <test-method status="SKIP" signature="untested_spec207_mustEnsureAllCallsOnItsSubscriptionTakePlaceFromTheSameThreadOrTakeCareOfSynchronization()" name="untested_spec207_mustEnsureAllCallsOnItsSubscriptionTakePlaceFromTheSameThreadOrTakeCareOfSynchronization" duration-ms="1" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Not verified using this TCK.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec207_mustEnsureAllCallsOnItsSubscriptionTakePlaceFromTheSameThreadOrTakeCareOfSynchronization(SubscriberWhiteboxVerification.java:246)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec207_mustEnsureAllCallsOnItsSubscriptionTakePlaceFromTheSameThreadOrTakeCareOfSynchronization(IdentityProcessorVerification.java:532)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
        </test-method>
        <test-method status="SKIP" signature="untested_spec206_mustCallSubscriptionCancelIfItIsNoLongerValid()" name="untested_spec206_mustCallSubscriptionCancelIfItIsNoLongerValid" duration-ms="0" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Not verified using this TCK.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec206_mustCallSubscriptionCancelIfItIsNoLongerValid(SubscriberWhiteboxVerification.java:240)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec206_mustCallSubscriptionCancelIfItIsNoLongerValid(IdentityProcessorVerification.java:527)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="SKIP" signature="untested_spec211_mustMakeSureThatAllCallsOnItsMethodsHappenBeforeTheProcessingOfTheRespectiveEvents()" name="untested_spec211_mustMakeSureThatAllCallsOnItsMethodsHappenBeforeTheProcessingOfTheRespectiveEvents" duration-ms="1" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Not verified using this TCK.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec211_mustMakeSureThatAllCallsOnItsMethodsHappenBeforeTheProcessingOfTheRespectiveEvents(SubscriberWhiteboxVerification.java:333)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec211_mustMakeSureThatAllCallsOnItsMethodsHappenBeforeTheProcessingOfTheRespectiveEvents(IdentityProcessorVerification.java:562)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="1" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
        </test-method>
        <test-method status="SKIP" signature="untested_spec212_mustNotCallOnSubscribeMoreThanOnceBasedOnObjectEquality_specViolation()" name="untested_spec212_mustNotCallOnSubscribeMoreThanOnceBasedOnObjectEquality_specViolation" duration-ms="0" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Not verified using this TCK.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec212_mustNotCallOnSubscribeMoreThanOnceBasedOnObjectEquality_specViolation(SubscriberWhiteboxVerification.java:339)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec212_mustNotCallOnSubscribeMoreThanOnceBasedOnObjectEquality_specViolation(IdentityProcessorVerification.java:567)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="1" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
        </test-method>
        <test-method status="SKIP" signature="untested_spec213_failingOnSignalInvocation()" name="untested_spec213_failingOnSignalInvocation" duration-ms="0" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Not verified using this TCK.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec213_failingOnSignalInvocation(SubscriberWhiteboxVerification.java:345)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec213_failingOnSignalInvocation(IdentityProcessorVerification.java:572)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
        </test-method>
        <test-method status="SKIP" signature="untested_spec304_requestShouldNotPerformHeavyComputations()" name="untested_spec304_requestShouldNotPerformHeavyComputations" duration-ms="1" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Not verified by this TCK.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Not verified by this TCK.
	at org.reactivestreams.tck.PublisherVerification.notVerified(PublisherVerification.java:1166)
	at org.reactivestreams.tck.PublisherVerification.untested_spec304_requestShouldNotPerformHeavyComputations(PublisherVerification.java:791)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec304_requestShouldNotPerformHeavyComputations(IdentityProcessorVerification.java:330)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
        </test-method>
        <test-method status="SKIP" signature="untested_spec301_mustNotBeCalledOutsideSubscriberContext()" name="untested_spec301_mustNotBeCalledOutsideSubscriberContext" duration-ms="0" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Not verified using this TCK.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec301_mustNotBeCalledOutsideSubscriberContext(SubscriberWhiteboxVerification.java:418)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec301_mustNotBeCalledOutsideSubscriberContext(IdentityProcessorVerification.java:590)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
        </test-method>
        <test-method status="SKIP" signature="untested_spec305_cancelMustNotSynchronouslyPerformHeavyCompuatation()" name="untested_spec305_cancelMustNotSynchronouslyPerformHeavyCompuatation" duration-ms="1" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Not verified by this TCK.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Not verified by this TCK.
	at org.reactivestreams.tck.PublisherVerification.notVerified(PublisherVerification.java:1166)
	at org.reactivestreams.tck.PublisherVerification.untested_spec305_cancelMustNotSynchronouslyPerformHeavyCompuatation(PublisherVerification.java:797)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec305_cancelMustNotSynchronouslyPerformHeavyCompuatation(IdentityProcessorVerification.java:335)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="SKIP" signature="untested_spec310_requestMaySynchronouslyCallOnNextOnSubscriber()" name="untested_spec310_requestMaySynchronouslyCallOnNextOnSubscriber" duration-ms="1" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Not verified using this TCK.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec310_requestMaySynchronouslyCallOnNextOnSubscriber(SubscriberWhiteboxVerification.java:442)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec310_requestMaySynchronouslyCallOnNextOnSubscriber(IdentityProcessorVerification.java:600)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
        </test-method>
        <test-method status="SKIP" signature="untested_spec311_requestMaySynchronouslyCallOnCompleteOrOnError()" name="untested_spec311_requestMaySynchronouslyCallOnCompleteOrOnError" duration-ms="0" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Not verified using this TCK.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec311_requestMaySynchronouslyCallOnCompleteOrOnError(SubscriberWhiteboxVerification.java:448)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec311_requestMaySynchronouslyCallOnCompleteOrOnError(IdentityProcessorVerification.java:605)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="1" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
        </test-method>
        <test-method status="SKIP" signature="untested_spec314_cancelMayCauseThePublisherToShutdownIfNoOtherSubscriptionExists()" name="untested_spec314_cancelMayCauseThePublisherToShutdownIfNoOtherSubscriptionExists" duration-ms="0" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Not verified using this TCK.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec314_cancelMayCauseThePublisherToShutdownIfNoOtherSubscriptionExists(SubscriberWhiteboxVerification.java:454)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec314_cancelMayCauseThePublisherToShutdownIfNoOtherSubscriptionExists(IdentityProcessorVerification.java:610)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="SKIP" signature="untested_spec315_cancelMustNotThrowExceptionAndMustSignalOnError()" name="untested_spec315_cancelMustNotThrowExceptionAndMustSignalOnError" duration-ms="0" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Not verified using this TCK.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec315_cancelMustNotThrowExceptionAndMustSignalOnError(SubscriberWhiteboxVerification.java:460)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec315_cancelMustNotThrowExceptionAndMustSignalOnError(IdentityProcessorVerification.java:615)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
        </test-method>
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="0" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
        </test-method>
        <test-method status="SKIP" signature="untested_spec316_requestMustNotThrowExceptionAndMustOnErrorTheSubscriber()" name="untested_spec316_requestMustNotThrowExceptionAndMustOnErrorTheSubscriber" duration-ms="0" started-at="2015-06-19T15:35:14Z" finished-at="2015-06-19T15:35:14Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Not verified using this TCK.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec316_requestMustNotThrowExceptionAndMustOnErrorTheSubscriber(SubscriberWhiteboxVerification.java:466)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec316_requestMustNotThrowExceptionAndMustOnErrorTheSubscriber(IdentityProcessorVerification.java:620)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
      </class>
    </test>
  </suite>
</testng-results>
